ouroboros-consensus/test-consensus/Test/Consensus/Mempool.hs

```diff
@@ -774,19 +850,25 @@ mempoolCapTestExpectedTrace mcts = go chunks
 -- | A data type containing 'StrictTVar's by which the two threads spawned by
 -- 'prop_Mempool_Capacity' can coordinate with each other.
 data MempoolCapTestEnv m = MempoolCapTestEnv
-  { mctEnvAddedTxs   :: StrictTVar m Word
-    -- ^ The number of transactions which have been added to the mempool.
-  , mctEnvRemovedTxs :: StrictTVar m Word
+  { mctEnvAddedBytes           :: StrictTVar m Word32
+    -- ^ The total number of bytes worth of transactions which have been added
+    -- to the mempool.
+  , mctEnvRemovedTxs           :: StrictTVar m Word
     -- ^ The number of transactions which have been removed from the mempool
     -- and added to the ledger.
+  , mctEnvBytesToBeAddedForCap :: StrictTVar m Word32
+    -- ^ The number of bytes that need to be added to the mempool to reach
+    -- full capacity.
```

Comment:
I don't think this adequately explains the overall purpose of this field. Will add better doc.